build/WebIDL.py

```diff
@@ -1,4 +1,4 @@
-# from http://mxr.mozilla.org/mozilla-central/source/dom/bindings/parser/WebIDL.py
+# from https://hg.mozilla.org/mozilla-central/file/tip/dom/bindings/parser/WebIDL.py
 # rev 501baeb3a034
 
```

Comment:
The revision also needs to be updated

Reply:
This is from [Emscripten repo](https://github.com/emscripten-core/emscripten/blob/e002f7fc5a5410575fc559181a1a7d135b10dba3/third_party/WebIDL.py), so it's better to comment the Emscripten commit.

Reply:
Added Emscripten Commit